### ISSUE TITLE
loss_memory.grad is none

### ISSUE THREAD

**Issue by 1090h2400** [haowang1992/DSN#2]
https://github.com/haowang1992/DSN/blob/0cb243182d20960b073f8c3bcc47c91611acc94f/src/train_cse_resnet_tuberlin_ext.py#L300-L302

.detach() causes loss_memory.grad to be none. It's not being used? （loss_memory
